server/graphql/core/src/loader/loader_registry.rs

```diff
@@ -268,6 +275,7 @@ pub async fn get_loaders(
         },
         async_std::task::spawn,
     ));
+    loaders.insert(temperature_breach_loader);
```

Comment:
Unrelated comment, IMHO we should just inline it like:

```rust
    loaders.insert(DataLoader::new(
        TemperatureBreachByIdLoader {
            connection_manager: connection_manager.clone(),
        },
        async_std::task::spawn,
    ));
```

easier to write and easier to read

Reply:
[c958358](https://github.com/openmsupply/open-msupply/pull/2326/commits/c958358f6a1b17ca88fd339ac5d73d114cb66e59)